gps_waypoints_server_node:
  max_start_str: 1054
  min_start_str: 986
  str_adj: 100
  skid_adj: 80
  str_mid: 969

  max_start_thr: 1080
  thr_adj: 100
  thr_slowest: 1140
  thr_mid: 1024

  goal_dist_thresh: 0.2
  goal_ang_thresh: 1.0
  x_track_error_start: 0.12
  x_track_repose_dist: 0.40

  pid_hdg_out_thresh: 0.7
  pid_x_out_thresh: 0.1

  hdg_p: 40.0
  hdg_i: 0.0
  hdg_d: 0.0
  cross_p: 150.0
  cross_i: 10.0
  cross_d: 0.0
  vel_p: 1.0
  vel_i: 0.0
  vel_d: 0.0

  use_heartbeat: False